Unix/base/paths.c

```diff
@@ -36,6 +36,7 @@
 #define HTTPRECVTRACEFILE LOGDIR "/omiserver-recv.trc"
 #define HTTPCLIENTSENDTRACEFILE LOGDIR "/omiclient-send.trc"
 #define HTTPCLIENTRECVTRACEFILE LOGDIR "/omiclient-recv.trc"
+#define NTLMCREDSFILE CONFIG_CREDSDIR "/ntlm"
```

Comment:
According to Paul, he would like the location of this file read from omiserver.conf. Talk to him and bring closure to this issue please.